README.md

```diff
@@ -0,0 +1,104 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-rails-back-end
+
+## Description
+
+> Since I familiar with Rails and React I put them together in a new kind of 'Hello World!' app. This exercise is going to have you create a React front-end with a Rails back-end and connect them to display a random message.
+
+
+
+## Learning objectives
+
+- Implement a connection between a Ruby on Rails back-end and React front-end.
+- Understand pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.
+
+
+## Getting Started
+
+To get a local copy for this project and running follow these simple example steps.
+
+## Link
+
+[react-front-end](https://github.com/kingqabil/hello-react-front-end/tree/Setup)
+
+[heroku](https://kingqabil-api.herokuapp.com/api/messages)
```

Comment:
- [ ] Great implementation👍🏽  However, the deployment link seems to be broken, after pushing to heroku, kindly ensure you run
```
heroku run rails db:migrate db:seed
```